WordPress/Classes/ViewRelated/Reader/Select Interests/ReaderInterestsDataSource.swift

```diff
@@ -68,12 +68,12 @@ class ReaderInterestsDataSource {
 
     /// Fetches the interests from the topic service
     public func reload() {
-        interestsService.fetchInterests(success: { interests in
-            self.interests = interests.map({ ReaderInterestViewModel(interest: $0)})
-        }) { (error: Error) in
+        interestsService.fetchInterests(success: { [weak self] interests in
+            self?.interests = interests.map { ReaderInterestViewModel(interest: $0)}
```

Comment:
super nitpicky but interests.map { ReaderInterestViewModel(interest: $0)} missing a space at the end between ) and }. I may add the space myself instead of bothering you with this